mteb/leaderboard/benchmark_selector.py

```diff
@@ -19,9 +21,20 @@
     ),
 ]
 """
+
+
+@dataclass
+class MenuEntry:
+    name: str | None
+    benchmarks: list[Benchmark]
+    open: bool
```

Comment:
Good addition. Maybe add default value to open to write it only once?

```suggestion
class MenuEntry:
    name: str | None
    benchmarks: list[Benchmark]
    open: bool = False
```

Reply:
I don't think it is a big deal, but will change it (depends on comments below)